pmm/pmm2-api-tests.groovy

```diff
@@ -73,72 +69,63 @@ pipeline {
                 sh 'git checkout ' + env.GIT_COMMIT_HASH
             }
         }
-        stage('Start staging') {
-            steps {
+        stage('API Tests Setup')
+        {
+            steps{
+                sh '''
+                    sudo curl -L https://github.com/docker/compose/releases/download/1.29.0/docker-compose-`uname -s`-`uname -m` | sudo tee /usr/local/bin/docker-compose > /dev/null
+                    sudo chmod +x /usr/local/bin/docker-compose
+                    sudo ln -sfn /usr/local/bin/docker-compose /usr/bin/docker-compose
+                    sudo docker-compose --version
+                    sudo docker build -t pmm-api-tests .
+                    git clone --single-branch --branch \${GIT_BRANCH_PMM_AGENT} https://github.com/percona/pmm-agent
+                    cd pmm-agent
+                    sudo PMM_SERVER_IMAGE=\${DOCKER_VERSION} MONGO_IMAGE=\${MONGO_IMAGE} MYSQL_IMAGE=\${MYSQL_IMAGE} POSTGRES_IMAGE=\${POSTGRES_IMAGE} docker-compose up -d
+                    cd ../
+                '''
                 script {
-                    if (!env.SERVER_IP) {
-                        runStaging(DOCKER_VERSION, CLIENT_VERSION, '--addclient=ps,1')
-                    }
-                    else
-                    {
-                        env.VM_IP = env.SERVER_IP
-                        env.PMM_URL = "http://admin:admin@${env.VM_IP}"
-                    }
+                    env.VM_IP = "127.0.0.1"
```

Comment:
Will the IP ever change?

Reply:
No, we are running PMM-Server on the worker directly now, so no change of IP

Reply:
I don't think it will change, since we are running server locally it will always be the same IP for pmm-server

Reply:
I mean if it never changes then we don't need a variable :)